### PR TITLE
[CSS/el] Greek CSS file

### PR DESCRIPTION
**Pull request by kbariotis**
Greek translation for the CSS

### CODE REVIEW

File: css-gr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,242 @@
+---
+language: css
+contributors:
+    - ["Kostas Bariotis", "http://kostasbariotis.com"]
+filename: css-gr.html.markdown
```
Comment (vendethiel) [inline css-gr.html.markdown]: please add `lang` key

Comment (kbariotis) [inline css-gr.html.markdown]: done

Comment (vendethiel) [inline css-gr.html.markdown]: you have to move it to the `el-gr` dir as well, thanks :)

Comment (kbariotis) [inline css-gr.html.markdown]: thank you.

**Comment by adambard** [comment]
Alright, header seems to be in order now. Thanks for your contribution!

**Pull request merged**